modules/gridBidAdapter.js

```diff
@@ -6,7 +6,7 @@ import { VIDEO, BANNER } from '../src/mediaTypes';
 const BIDDER_CODE = 'grid';
 const ENDPOINT_URL = '//grid.bidswitch.net/hb';
 const TIME_TO_LIVE = 360;
-const RENDERER_URL = '//cdn.adnxs.com/renderer/video/ANOutstreamVideo.js';
+const RENDERER_URL = '//acdn.adnxs.com/video/outstream/ANOutstreamVideo.js';
```

Comment:
Do we know if `GridBidAdapter` and `TrustxBidAdapter` both want to move to latest version of outstream renderer ?

Reply:
Hi @TheMediaGrid, @PWyrembak , 
Your adapters are using an older version of `outstream production url`, so I went ahead and updated it to the new url. Can you please check and approve this change by reverting to this comment or let me know your feedback if have any other opinion.

Reply:
@sumit116 @jaiminpanchal27 Approved for trustxBidAdapter.js.
Is it a simple upgrade? Should we expect any performance changes on outstream front?

Reply:
@sumit116 we have finished testing of the new url
Approved for merge